src/Digdir.Domain.Dialogporten.Application/Features/V1/ServiceOwner/Dialogs/Commands/Create/CreateDialogCommandValidator.cs

```diff
@@ -444,7 +445,7 @@ public CreateDialogDialogActivityDtoValidator(
     }
 }
 
-internal sealed class CreateDialogDialogTransmissionActorDtoValidator : AbstractValidator<TransmissionSenderActorDto>
+internal sealed class CreateDialogDialogTransmissionActorDtoValidator : AbstractValidator<ActorDto>
```

Comment:
Hmm, dette er et issue... Vi burde egentlig ha **en** validator per `ActorDto`. Nå lager vi flere gjennom hele systemet, og hvilken abstraksjon som vinner når vi ber om en `IValidator<ActorDto>` er anybody's guess.

Reply:
Ser ut som det ikke var noe validering på Enduser ActorDto, er vell nok med å lage den kun validator for SO?